scrape/apps_test.go

```diff
@@ -0,0 +1,75 @@
+package scrape
+
+import (
+	"net/http"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+)
+
+func Test_AppRestrictionsEnabled(t *testing.T) {
+	client, mux, cleanup := setup()
+	defer cleanup()
+
```

Comment:
Would be nice to make these table driven tests:

```
tests := []struct {
	description    string
	testFile string
	org string
	want bool
}{
  {
     description: "return true for enabled orgs",
     testFile: "access-restrictions-enabled.html",
     org: "e",
     want: true,
  },
  ....
}
for _, tt := range tests {
    t.Run(tt.description, func(t *testing.T) {
        client, mux, cleanup := setup()
        defer cleanup()
        mux.HandleFunc(fmt.Sprintf("/organizations/%v/settings/oauth_application_policy", tt.org), func(w http.ResponseWriter, r *http.Request) {
		copyTestFile(w, tt.file)
	})
        got, err := client.AppRestrictionsEnabled(tt.org)
	if err != nil {
		t.Errorf("AppRestrictionsEnabled() returned err: %v", err)
	}
	if want := true; got != want {
		t.Errorf("AppRestrictionsEnabled(); got %t, want %t", got, want)
	}
    })
}
```

Reply:
done